CommunityToolkit.Mvvm.SourceGenerators/ComponentModel/ObservableValidatorValidateAllPropertiesGenerator.Execute.cs

```diff
@@ -108,20 +108,56 @@ public static RecipientInfo GetInfo(INamedTypeSymbol typeSymbol, ImmutableArray<
         /// <summary>
         /// Gets the head <see cref="CompilationUnitSyntax"/> instance.
         /// </summary>
+        /// <param name="isDynamicallyAccessedMembersAttributeAvailable">Indicates whether <c>[DynamicallyAccessedMembers]</c> should be generated.</param>
         /// <returns>The head <see cref="CompilationUnitSyntax"/> instance with the type attributes.</returns>
-        public static CompilationUnitSyntax GetSyntax()
+        public static CompilationUnitSyntax GetSyntax(bool isDynamicallyAccessedMembersAttributeAvailable)
         {
+            int numberOfAttributes = 5 + (isDynamicallyAccessedMembersAttributeAvailable ? 1 : 0);
```

Comment:
What is this 5 determined by? Is it possible to get this number programmatically, rather than hardcoding?
If not, this should probably have a comment on it.

Reply:
Yeah I see that, just wondering if hardcoding is a good idea